spec/models/comment_spec.rb

```diff
@@ -0,0 +1,8 @@
+require 'rails_helper'
+
+RSpec.describe Comment, type: :model do
+  it 'should not create comment without text' do
+    comment = Comment.create(post_id: 8, author_id: 3, text: nil)
+    expect(comment).to_not be_valid
+  end
+end
```

Comment:
[OPTIONAL] Following the project requirements, and as requested by the previous reviewer, any custom methods in the models should have a corresponding unit test. It would be better to create tests for the update_comments_counter method in the Comment method, update_counter method in the Post model, and update_likes_counter in the Like model.